Remove old email alias (#4598)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/devices/index.md

<ApiLifecycle access="ie" />

> > **Note:** This feature is only available as a part of Okta Identity Engine. Please [contact support](https://support.okta.com/) for further information.

Reviewer: ```suggestion
> **Note:** This feature is only available as a part of Okta Identity Engine. [Contact support](https://support.okta.com/)  for information on the Identity Engine.
```

Permanently deletes a Device that is in `DEACTIVATED` status. The Device can be transitioned to `DEACTIVATED` status using the [deactivate](#deactivate-device) API.

> This deletion is destructive and deletes all the profile data related to the device. Once deleted, device data can't be recovered. A Device that isn't in a `DEACTIVATED` state raises an error if a delete operation is attempted.

Reviewer: ```suggestion
This deletion is destructive and deletes all the profile data related to the device. After it's deleted, device data can't be recovered. A Device that isn't in a `DEACTIVATED` state raises an error if a delete operation is attempted.
```
